arm/Microsoft.ContainerService/managedClusters/deploy.bicep

```diff
@@ -328,10 +334,10 @@ var diagnosticsMetrics = [for metric in diagnosticMetricsToEnable: {
 
 var identityType = systemAssignedIdentity ? 'SystemAssigned' : (!empty(userAssignedIdentities) ? 'UserAssigned' : 'None')
 
-var identity = identityType != 'None' ? {
```

Comment:
Why did you remove that?

Reply:
The `identity` block should not be null or empty. You have to set a value for `type`.